tests/acceptance/15_control/02_agent/agentaccess.cf

```diff
@@ -0,0 +1,17 @@
+body common control
```

Comment:
Please add a short comment what this tests (`agentaccess`) and how.

Reply:
done. for some reason not auto-resolved in the review.